app/models/user.rb

```diff
@@ -91,6 +97,21 @@ def password_reset_expired?
   end
 
   def feed
-    microposts.created_at_desc
+    Micropost.where("user_id IN (?) OR user_id = ?", following_ids, id)
```

Comment:
`.where("user_id IN (?) OR user_id = ?", following_ids, id)` viết scope nào

Reply:
Đã fix